io_scene_godot/converters/material.py

```diff
@@ -47,21 +48,37 @@ def export_material(escn_file, export_settings, material):
             resource_id = escn_file.add_external_resource(ext_mat, material)
         return "ExtResource({})".format(resource_id)
 
+    resource_id = generate_material_resource(
+        escn_file, export_settings, material, has_uv
+    )
+    return "SubResource({})".format(resource_id)
+
+
+def generate_material_resource(escn_file, export_settings, material, has_uv):
+    """Export blender material as an internal resource"""
```

Comment:
In wonder if we should detect if cycles or blender internal is being used for the renderer?

Reply:
I didn't notice blender internal engine material also can be edited use nodes, I think you are right @sdfgeoff , it needs a check for what engine is rendering.

For blender2.8, I have seen they changed add-on APIs, I think it's not likely to have compatible version of exporter both works on blender2.8 and older versions. So it makes the situation simpler, we can always default to use node tree for blender 2.8